app/models/icla_signature.rb

```diff
@@ -28,7 +28,7 @@ class IclaSignature < ActiveRecord::Base
 
   # Callbacks
   # --------------------
-  before_create -> (record) { record.signed_at = Time.now }
+  before_create -> (record) { record.signed_at ||= Time.now }
```

Comment:
I realize it was already like this, but we should be using `Time.zone.now` everywhere instead of `Time.now`.